src/subsystem/can/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+add_library(rover_can rover_can.hpp constants.hpp commands.hpp commands.cpp)
+
+target_include_directories(rover_can PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
+
+# If ONBOARD_CAN_BUS has been manually defined (or is cached), then don't override it with auto-detection
+if (NOT DEFINED ONBOARD_CAN_BUS)
+	if (${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
+		set(ONBOARD_CAN_BUS OFF CACHE BOOL "Compile subsystem with Linux CAN bus libraries")
```

Comment:
This is supposed to be set to ON. Complete a rebuild to test on both Linux and non-Linux.